Documentation/ApiOverview/Deprecation/Index.rst

```diff
@@ -70,7 +70,7 @@ calls to the affected functions.
 Deprecate functions in extensions
 =================================
 
-Functions that will be removed in future versions of your extension should be marked as deprecated by both the
+Functions, that will be removed in future versions of your extension, should be marked as deprecated by both the
```

Comment:
Same here.

Reply:
No, this is  a differnet case.

"Functions ... should be marked as deprecated".

This would be wrong. Not all functions should be marked as deprecated.